docs/src/TextField.doc.js

```diff
@@ -224,15 +225,21 @@ function Example(props) {
         id="tags"
         label="Emails"
         onChange={({ value }) => {
-          const tagInput = value.split(/[ ,;]+/);
+          const tagInput = value.split(/[\\s,;]+/);
```

Comment:
not super opinionated on this but the example could read easier if we move out callbacks


![image](https://user-images.githubusercontent.com/10593890/102395097-ec2e4900-3f97-11eb-9e55-ee8103be4bca.png)